up-l3/utwin/v2/README.adoc

```diff
@@ -20,21 +20,21 @@ SPDX-License-Identifier: Apache-2.0
 
 == Purpose
 
-In uProtocol, uEs rely on other uEs that expose services, named uServices. These uServices publish information, named events, on Topics, through the implementations of uProtocol. These events can be considered to represent updates to the uService _state._ The collection of last published event of each Topic of a uService therefore represents the current "public state" of the uService (public because some more internal states might exist without being exposed externally).
+In uProtocol, uEntities (might be abbreviated as `uE`, especially in diagrams) rely on other uEntities that expose services, named uServices. These uServices publish information, named events, on Topics, through the implementations of uProtocol. These events can be considered to represent updates to the uService _state._ The collection of last published event of each Topic of a uService therefore represents the current "public state" of the uService (public because some more internal states might exist without being exposed externally).
```

Comment:
```suggestion
In uProtocol, uEntities (might be abbreviated as *uE*, especially in diagrams) rely on other uEntities that expose services, named uServices. These uServices publish information, named events, on Topics, through the implementations of uProtocol. These events can be considered to represent updates to the uService _state._ The collection of last published event of each Topic of a uService therefore represents the current "public state" of the uService (public because some more internal states might exist without being exposed externally).
```